.circleci/config.yml

```diff
@@ -99,15 +99,19 @@ workflows:
   build-and-publish:
     jobs:
       - build:
+          context: maven-central-publish
```

Comment:
we should only add this on the release job. If the plugin requires this when publish isn't invoked, we should update that on the plugin side (as users will not have these values set)

Reply:
Raised the following PR: https://github.com/hypertrace/hypertrace-gradle-publish-plugin/pull/16